### ISSUE TITLE
[desktop] Support navigating UI by tab

### ISSUE THREAD

**Issue by twiss** [codexa/firetext#212]
Basically, make sure everything has a logical tabindex and a visible outline on `:focus`.

Spin-off from https://github.com/codexa/firetext/issues/206#issuecomment-62367711

**Comment by ferndot** [comment]
This is a good first-fix issue!

**Comment by twiss** [comment]
Yes. We should also make sure you can leave the editor's focus, e.g. in WebODF, tab to focus next won't work.
